docs/install/install-edex.md

```diff
@@ -275,3 +274,114 @@ This file specifies configuration and runtime parameters. If you are pulling in
 ```
 rpm -qa | grep awips2-edex
 ```
+
+## Uninstalling EDEX
+These are instructions to manually uninstall EDEX. However, the [`awips_install.sh`](#1-install-edex) script will do all of these steps for you if you are installing a newer version of EDEX.
+
+**1. Make sure all EDEX processes are stopped**
+```
+sudo edex stop
+sudo edex status
+
+[edex status]
+ postgres    :: not running
+ pypies      :: not running
+ qpid        :: not running
+ EDEXingest  :: not running
+ EDEXgrib    :: not running
+ EDEXrequest :: not running
+ ldmadmin    :: not running
+```
+
+**2. Backup any important configuration files that you may want to reference**
+
+Here are some possible important directories/files to backup:
+
+```
+/awips2/database/data/pg_hba.conf
```

Comment:
Do you think it would be useful to put "/*" at the end of directories, to show that we mean the whole directory?
Ie.  /awips2/ldm/\*